packages/events/defaultSubscribers/migrate/Messages.js

```diff
@@ -220,7 +220,7 @@ class Messages {
       deployed: () => {
         let stopText;
         if (reporter.blockSpinner) {
-          reporter.blockSpinner.stop();
+          reporter.blockSpinner.remove();
```

Comment:
Does `blockSpinner.remove()` have a meaningful implication to `stopText`? It also seems weird that, in the pre-commit `stop()` is invoked before setting `stopText`

Reply:
`stopText` is unrelated to the spinner - I didn't write that code, but it appears to be used for logging output.

Reply:
oh! I see. Thanks!